apps/02RockPaperScissors.js

```diff
@@ -7,10 +7,43 @@ var rl = readline.createInterface({
   output: process.stdout
 });
 
+  // Hand one is a variable that will just have the word Rock Paper or Scissor in it!
+
+  //Spec 1 - If it's a tie, return "It's a tie!" It must be exactly "It's a tie!" for the tests to pass
+
 
 function rockPaperScissors(hand1, hand2) {
```

Comment:
At the top of your PigLatin app, you took word and immediately turned it to word.toLowerCase(), I'm curious why you didn't do that with hand1 and hand2 here.